packages/ui/tailwind-utils-config/components/button.ts

```diff
@@ -180,6 +188,22 @@ export default {
         height: 'var(--cn-icon-size-sm)'
       }
     },
+    '&:where(.cn-button-icon-only.cn-button-2xs)': {
```

Comment:
@ankormoreankor can we remove the button size changes in this as discussed? I am making these changes in my local and it has more additional changes to this, added after discussing with @cxramina.